app/src/main/java/ch/epfl/sdp/appart/user/AppUser.java

```diff
@@ -19,11 +20,9 @@
     private String phoneNumber;
     private long age;
     private Gender gender;
-    private String profilePicture;
+    private String profileImage;
```

Comment:
Nitpick comment but profileImage refers to the path or the name ? Maybe profileImageName/profileImagePath as name ?

Reply:
Good point. It's worth it to make it very clear. Actually this attribute of user contains both the path and the image name. This is the reason why deleteImage has signature:
`deleteImage(String imagePathAndName)`

the method can be called (in the user image case) as:
`deleteImage(this.getProfileImage())`

I thus suggest for coherence that the user attribute becomes :
`private String profileImagePathAndName`